lib/commands/star.js

```diff
@@ -0,0 +1,20 @@
+var log = require('loglevel');
+var core = require('../core');
+var h = require('../helper');
+
+var cmd = {
+  command: 'star <keyword>',
+  desc:    'star problem by name or index'
+};
+
+cmd.handler = function(argv) {
+  core.getProblem(argv.keyword, function(e, problem) {
+    if (e) return log.fail(e);
+    core.starProblem(problem, function(e) {
+      if (e) return log.fail(e);
+      log.info(h.prettyText(' Starred', true));
```

Comment:
Thanks dude, overall LGTM. Just wondering what if run 'lc star 1' twice, will it be un-starred in the end?